### DISCUSSION TITLE
Build Input Help

### DISCUSSION THREAD

**Discussion by bobby31887** [manna-harbour/miryoku#86] [Q&A] [answered]
Can someone please help me with this error message?  I'm not sure what I'm doing wrong.  I want to build the ferris/0_2/compact bin file with everything else set as default.  The link below shows the error messages and you can see I tried with different keyboards like the corne and still got the same error message.

https://github.com/bobby31887/miryoku_qmk/actions/workflows/build-inputs.yml 

![image](https://user-images.githubusercontent.com/80011808/170852254-f2b51430-d5af-4ae6-b4c7-487684492cf8.png)
![image](https://user-images.githubusercontent.com/80011808/170852264-36d12df9-39a7-40fe-a3c2-b82c98c0b4ff.png)

Thank you!

**Comment by manna-harbour** [comment, marked as answer]
You just need to update your fork. The version you have references the workflow on my fork by branch name, but that's no longer compatible with what's in your fork. Later versions used a commit ref instead of the branch name to avoid this problem. And the current version uses the workflow in the local fork instead. But for the version you have you'll need to update. 

[Here's a sample build for ferris/0_2/compact.](https://github.com/manna-harbour/miryoku_qmk/actions/runs/2403355670)

**Reply by bobby31887** [reply]
Thanks for the help.
